src/Apps/Order/__tests__/OrderApp.test.tsx

```diff
@@ -12,14 +12,13 @@ import { OrderApp } from "../OrderApp"
 import {
   BuyOrderPickup,
   BuyOrderWithShippingDetails,
-  mockResolver,
   OfferOrderWithShippingDetails,
   OfferWithTotals,
   UntouchedBuyOrder,
   UntouchedOfferOrder,
 } from "Apps/__tests__/Fixtures/Order"
 import { MockBoot } from "DevTools"
-import { createMockNetworkLayer } from "DevTools/createMockNetworkLayer"
+import { createMockNetworkLayer2 } from "DevTools/createMockNetworkLayer"
```

Comment:
Can you explain this change?

Reply:
`createMockNetworkLayer2` was a re-implementation of `createMockNetworkLayer` with a few changes

- more ergonomic api for supplying mock data (result-based rather than type-based)
- much better error reporting
- id field consolidation

This last one is the one that matters in this test file. Since the fake-stitched `Order` type [gained an `__id` field in MP recently](https://github.com/artsy/metaphysics/commit/cb3e7d73b4c3366a34471b186abd79fa74340636#diff-ca178747ca9568ddd30e7dffd824b873R8232), it changed how the queries on `Order` were being generated. Our custom version of relay aliases `id` to `__id` when there's no explicit `__id` field. (`__id` is what relay uses for indexing data in the cache). Now that there is an explicit `__id` field on `Order`, the aliasing disappeared.

So now I think what must have been happening with the old version of `createMockNetworkLayer` was that the aliasing information in the queries told it to resolve the `id` fields from the fixtures, but once the aliasing was converted into direct lookups on `__id` it didn't know about that relationship any more. So we had these `id` fields in the fixtures but the resolution logic was looking for `__id` fields only.

`createMockNetworkLayer2` has this id-aliasing knowledge built-in and was able to take direct lookups on `__id` and resolve the value in the `id` field.